pkg/roachprod/roachprod.go

```diff
@@ -808,10 +808,10 @@ func updatePrometheusTargets(ctx context.Context, l *logger.Logger, c *install.S
 					l.Errorf("error getting the port for node %d: %v", index, err)
 					return
 				}
-				nodeInfo := fmt.Sprintf("%s:%d", v.PublicIP, desc.Port)
+				nodeInfo := fmt.Sprintf("%s:%d", v.PrivateIP, desc.Port)
```

Comment:
Good catch! GCE roachprod clusters are reachable to prometheus via private ips, so we save on egress. We might reconsider at a later time in the context of multi-cloud clusters; for now, those are rare.